src/cmap/connection_pool.ts

```diff
@@ -90,6 +90,8 @@ export interface ConnectionPoolOptions extends Omit<ConnectionOptions, 'id' | 'g
   waitQueueTimeoutMS: number;
   /** If we are in load balancer mode. */
   loadBalanced: boolean;
+  /** @experimental */
```

Comment:
do we have a preference for marking this experimental or internal? when we release the next version, we may want to omit this item from the feature list or call it out as internal in the release notes, since users can't really set it directly anwyay

Reply:
The difference will be that internal will get removed from the public types, so I think that would be the better annotation if this isn't publicly configurable